### PR TITLE
feat: overview charts and protocols

### PR DESCRIPTION
**Pull request by frimoldi**
image

### CODE REVIEW

File: src/pages/Overview/Overview.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        value: Number(utils.formatUnits(tvc.value, 6)),
+      })
+      tvlChartData.push({
+        name: DateTime.fromMillis(timestamp * 1000).toFormat("M/d"),
+        value: Number(utils.formatUnits(tvl.value, 6)),
+      })
+      riskChartData.push({
+        name: DateTime.fromMillis(timestamp * 1000).toFormat("M/d"),
+        value: Number(utils.formatUnits(tvc.value, 6)) / Number(utils.formatUnits(tvl.value, 6)),
+      })
```
Comment (rcstanciu) [inline src/pages/Overview/Overview.tsx:69]: At the moment we must traverse both datasets to compute the risk factor data, which does not allow for refactoring. I think this data can be easily computed either in the indexer or at the db leve (with something like a materialised view)

**Comment by rcstanciu** [comment]
If the content is bigger than viewport height, the content is pushed somewhere under the header, without the possibility of scrolling upwards to see it:
Screenshot 2022-04-19 at 16 32 32

Might be related to the Boxes that are not centered anymore (they are a bit lower):
Screenshot 2022-04-19 at 16 35 00

File: src/pages/Overview/Overview.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+          tvl = j > 0 ? tvlData[j - 1] : tvlData[j]
+          i++
+        } else {
+          tvc = i > 0 ? tvcData[i - 1] : tvcData[i]
+          j++
+        }
+      } else {
+        i++
+        j++
+      }
```
[the remark this answers is not in the document]
Comment (frimoldi) [reply src/pages/Overview/Overview.tsx:54]: yeah, this is a god point. I've already created a separate component for the chart itself, abstracting recharts. But I think another layer of abstraction is needed here.

File: src/pages/Overview/Overview.tsx (hunk 1 above)
Comment (frimoldi) [reply src/pages/Overview/Overview.tsx:69]: I'll create a Trello card for it.

**Comment by frimoldi** [comment]
@rcstanciu these should all be fixed now.

**Review by rcstanciu — APPROVED** [review]
Issues fixed, looks good to me

**Pull request merged**